programs/survival/elytra_swap.sc

```diff
@@ -0,0 +1,17 @@
+// By right-clicking with an elytra or any chestplate this script will swap the items
+// By: IceWolf23
+
+global_swappable_items = {'netherite_chestplate','diamond_chestplate','iron_chestplate','golden_chestplate','chainmail_chestplate','leather_chestplate','elytra'};
```

Comment:
Initializing a map with a series of entries turns them into keys with values of `null`. That won't work with line 9, will it? I suggest initializing them with a non-null value or going back to the tried and true list form.

Reply:
I changed it to use a map, forgot to change the bottom bit...